### PR TITLE
Refactor: Relocate Validation Alert and Fix Missing Report Type - 973, 975

### LINKED ISSUE #973: LCFS - BUG - Validation alert, screen real estate
**Describe the Bug**
Can we reserve the screen real estate for the validation/error alert? Presently when the bar disappears the table bumps up. This would be on all schedules and forms with validation.

Relocate alert to directly under the table (above New row and Save button).

Image

**Expected Behaviour**

### PR DESCRIPTION
**Pull request by hamed-valiollahi**
This PR relocates the validation alert directly under the table to prevent UI shifting and fixes the missing compliance report type text in the index page.

Closes #973, #975

### CODE REVIEW

File: frontend/src/views/ComplianceReports/components/_schema.jsx [hunk 1]
```diff
@@ -22,7 +22,7 @@ export const reportsColDefs = (t, bceidRole) => [
     field: 'type',
     headerName: t('report:reportColLabels.type'),
     flex: 2,
-    valueGetter: ({ data }) => data.type?.description || ''
+    valueGetter: () => 'Compliance report'
```
Comment (dhaselhan) [inline frontend/src/views/ComplianceReports/components/_schema.jsx:25]: suggestion: Update this to use the translation file

**Review by dhaselhan — CHANGES_REQUESTED** [review]
Small change for translation. I believe we should be putting all UI strings there, but correct me if I'm wrong.

**Comment by hamed-valiollahi** [comment]
> Small change for translation. I believe we should be putting all UI strings there, but correct me if I'm wrong.

I’ve moved all the UI strings to the locale file. Thanks for reviewing the code, Daniel! 👍

**Review by dhaselhan — APPROVED** [review]
LGTM 👍

**Pull request merged**